tdms/tests/system/test_fdtd.py

```diff
@@ -26,13 +24,14 @@ def test_fs():
     """
 
     # misleading filename here
-    run_tdms("arc_example_fdtd/in_pstd_fs.mat", "fdtd_fs_output.mat")
+    run_tdms(
```

Comment:
Do I understand correctly that this is now always running the tests with finite differences only?

Reply:
Yes, exactly. The finite difference code results in a different reference output. So it never makes sense to run over this file without nor the others with.